lib/carrierwave/storage/dropbox.rb

```diff
@@ -44,8 +44,12 @@ def initialize(uploader, config, path, client)
         end
 
         def url
-          metadata, result = @client.get_temporary_link("/#{@path}")
-          result
+          begin
+            metadata, result = @client.get_temporary_link("/#{@path}")
+            result
+          rescue ::Dropbox::ApiError
+            ""
```

Comment:
Thoughts on returning `nil` instead, to better align with `ActiveRecord` semantics? Allows for consistent behavior with conditionals, as `""` is truthy (ie, `if my_model.image.url...else...end` should execute the `else` block).